lambdas/db-provision-user-database/tests/test-index.js

```diff
@@ -15,6 +15,7 @@ test.before(async (t) => {
         username: localStackConnectionEnv.PG_USER,
         password: localStackConnectionEnv.PG_PASSWORD,
         database: localStackConnectionEnv.PG_DATABASE,
+        disableSSL: 'true',
```

Comment:
So we're disabling ssl in tests because that's not explicitly what we want to test and presumably requires a cert and some setup?

Reply:
Exactly that - we're not trying to test that knex connects via SSL properly, we're using the db as a stand in instead of mocking to test the logic.